docs/en/operation/Metrics-System.md

```diff
@@ -121,6 +132,10 @@ scrape_configs:
       metrics_path: '/metrics/prometheus/'
       static_configs:
       - targets: [ '<WORKER_HOSTNAME>:<WORKER_WEB_PORT>' ]
+  - job_name: "alluxio standalone fuse"
+      metrics_path: '/metrics/prometheus/'
+      static_configs:
+         - targets: [ '<FUSE_WEB_HOSTNAME>:<FUSE_WEB_PORT>' ]
```

Comment:
Would you please to reduce 4 space to align the target element to the previous element `static_configs`.

Reply:
Fixed, thanks for catching the problem, PTAL